pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[project]
+name = "qoqo_myqlm"
+version = "0.4.3"
+description = "An inteface to execute circuits from QOQO to MYQLM"
+authors = [{name="HQS Quantum Simulation GmbH", email="info@quantumsimulations.de"}]
+readme = "README.md"
+requires-python = ">3.8"
+dependencies = [
+    "qoqo>=1.0.0",
```

Comment:
Should this be >=1.2?

Reply:
It could, I just copied the one in setup.py. Wouldn`t this leave more room for versions? (if the repo is working just fine with 1.0.0 and higher)

Reply:
I believe we always want to be using the latest version, but that's a good point, we should ask Nico

Reply:
changed to >=1.2